schema.sql

```diff
@@ -15,4 +20,17 @@ CREATE TABLE author(
     id INT NOT NULL PRIMARY KEY,
     first_name varchar(50),
     last_name varchar(50)
+);
+
+CREATE TABLE labels (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR(255) NOT NULL,
+  color VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE books (
+  id INTEGER REFERENCES item(id) PRIMARY KEY,
```

Comment:
I would suggest to specify the field name with the type of table `book_id` this will help to avoid ambiguous data error. :slightly_smiling_face: